models/labels/trader_portfolios/ethereum/trader_portfolios_ethereum.sql

```diff
@@ -0,0 +1,45 @@
+{{
+    config(
+        alias='trader_portfolios_ethereum',
+    )
+}}
+
+with
+ trader_portfolios as (
+    select
+        sum(amount_usd) as portfolio_value_usd,
+        wallet_address as address
+    from {{ ref('balances_ethereum_erc20_day') }}
```

Comment:
I am a bit concerned about using this table as a foundation because of the fact that I fear there are untamed bugs here. For example in the latest balances table our settlement has 8 negative balances

https://dune.com/queries/1761899

Of course it is only natural to depend on this, but the underlying table will need to be debugged eventually.

Reply:
Note that numerical overflow is a very likely candidate for these bugs... but it may not be the only problem.

![Screenshot 2022-12-15 at 11 23 26](https://user-images.githubusercontent.com/11778116/207834996-7ac71fb4-b1a7-459d-860c-03ee7ef83a0c.png)

Reply:
I agree with your concerns and realize the flaws of the table. I'll keep them in mind when doing any analysis using this label. I've added an `amount > 0` condition to ensure I don't include these errors in the label calculation.

Reply:
That is a good start, but just because we exclude negatives does not mean that we are excluding the bad accounting. There could be positive amounts that are still incorrect due to this very same bug...